tests/framework/utils/utils.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
How about naming this retry.go? I'd be happy if nothing in rook were called `utils`, it's so generic.